patches/v2.0/0000-cover-letter.patch

```diff
@@ -1,7 +1,7 @@
 From 7c60851daae39815fa696fc2265e838b1913f7ac Mon Sep 17 00:00:00 2001
 From: Wentao Zhang <zhangwt1997@gmail.com>
```

Comment:
In the sent version 2 I accidentally included my personal email here in the cover letter. I suppose it should not create a "Signed-off-by chain" issue as it's not a patch? I will pay attention in future updates.